test/GGenerator.hs

```diff
@@ -6,13 +6,14 @@
 
 module Main where
 
-import           Universum
+import           Universum  hiding (writeFile)
```

Comment:
`writeFile` from `universum` is just:

```haskell
writeFile :: MonadIO m => FilePath -> Text -> m ()
writeFile a b = liftIO (Data.Text.IO.writeFile a b)
```

So I don't see reasons to hide this `writeFile` and use it directly from `Data.Text.IO`.

Reply:
I should have looked at the `writeFile` there. I just used the `Text.IO` one and fixed the resulting error.